cloudinit/sources/helpers/vmware/imc/guestcust_util.py

```diff
@@ -265,6 +265,17 @@ def get_data_from_imc_raw_data_cust_cfg(cust_cfg):
             )
             return (None, None, None)
 
+        try:
+            logger.debug("Safeyaml loads md tocheck if it's valid")
```

Comment:
usually you would log something like: "Validating meta data"

Reply:
And corrected the Proposed Commit Message format.